recipe/meta.yaml

```diff
@@ -33,6 +33,7 @@ requirements:
 
 test:
   requires:
+    - diffutils  # [not win]
```

Comment:
Needs to be under `build` since the CTest tests are running as part of the build. The only things running during the conda-build `test` phase are the lines under `command:` (e.g. `codes_info`).

Reply:
Thanks Ryan (@dopplershift) - that seems to have fixed it - well spotted!